tools/packager/main.go

```diff
@@ -35,16 +37,28 @@ func main() {
 	defer f.Close()
 	zw := gzip.NewWriter(f)
 	defer zw.Close()
-	tw := tar.NewWriter(zw)
+	tw := archive.NewNormalizingTarWriter(tar.NewWriter(zw))
+	tw.WithUID(0)
+	tw.WithGID(0)
 	defer tw.Close()
 
+	descriptorInfo, err := os.Stat(descriptorPath)
+	if err != nil {
+		fmt.Printf("Failed stat descriptor file at path %s: %s", descriptorPath, err)
+		os.Exit(3)
+	}
+	if err := archive.AddFileToArchive(tw, "lifecycle.toml", descriptorInfo); err != nil {
+		fmt.Printf("Failed to write descriptor to arichive: %s", err)
+		os.Exit(4)
+	}
+
 	if err := os.Chdir(filepath.Dir(inputDir)); err != nil {
 		fmt.Printf("Failed to switch directories to %s: %s", filepath.Dir(inputDir), err)
-		os.Exit(3)
```

Comment:
Are these being changed to spec or just incremented because some other code has changed?

Reply:
This is essentially just a script for our internal use while packaging the lifecycle, totally unspecified. `tools` is actually it's own module right now, although I think an internal package is probably more appropriate in the future.